tests/DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.Faker/Helpers/EstablishmentTypePicker.cs

```diff
@@ -0,0 +1,51 @@
+using DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.Models.Gias;
+
+namespace DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.Faker.Helpers;
+
+public interface IEstablishmentTypePicker
+{
+    public string[] GetEstablishmentTypes(GiasEstablishment establishment);
+}
+
+public class EstablishmentTypePicker : IEstablishmentTypePicker
+{
+    private readonly string[] _secondaryAllThroughPhaseNames =
+    {
+        "Secondary", "Middle Deemed Secondary", "16 Plus", "Not applicable", "All Through", "All-Through",
```

Comment:
This one should be updated to remove the surplus "all through" types - look at `Data.GiasPhaseNames` for the final version :-)